src/assets/scripts/client/ui/UiController.js

```diff
@@ -381,6 +393,7 @@ class UiController {
      */
     setupHandlers() {
         this.onAirportChangeHandler = this.onAirportChange.bind(this);
+        this.onChatLogDurationChangeHandler = this.onChatLogDurationChange.bind(this);
```

Comment:
Well done!! This trips folks up sometimes cause it's not something folks see a lot these days.